packages/react-instantsearch/src/widgets/Hits.js

```diff
@@ -1,11 +1,9 @@
-/**
- * @module widgets/Hits
```

Comment:
Good call removing those. I expected to have a use for them, but in the end it's doesn't work.